src/transformers/pipelines/__init__.py

```diff
@@ -773,6 +776,10 @@ def pipeline(
     load_feature_extractor = type(model_config) in FEATURE_EXTRACTOR_MAPPING or feature_extractor is not None
     load_image_processor = type(model_config) in IMAGE_PROCESSOR_MAPPING or image_processor is not None
 
+    # Just keep `load_image_processor`
+    if load_image_processor and load_feature_extractor:
+        load_feature_extractor = False
```

Comment:
This is arguably a bit breaking, if one want to access `self.feature_extractor`. I can remove this block if @sgugger also agree it is better/should.

Reply:
Mmm what happens for a model that mixes image and sound then?

Reply:
Yeah, good question. I don't think we have this use case yet, but as I also have a doubt mentioned above, I can simply remove this block to avoid any potentially issue.

Reply:
Yes let's remove this and keep this as simple as possible (it's already quite tortuous)

Reply:
OK